Paintroid/src/main/res/layout/pocketpaint_layout_top_bar.xml

```diff
@@ -79,6 +79,7 @@
         android:id="@+id/pocketpaint_toolbar"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        app:popupTheme="@style/MoreOptionsMenuBackground"
```

Comment:
you can remove this if you move the `MoreOptionsMenuBackground` style directly into the base theme